PublicationsRouter/v2/sword-out/pubrouter-xml/pubrouter.xsd

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+    targetNamespace="http://pubrouter.jisc.ac.uk/rioxxplus"
+    xmlns:rioxxterms="http://www.rioxx.net/schema/v2.0/rioxxterms/"
+    xmlns:ali="http://ali.niso.org/2014/ali/1.0"
+    xmlns:dcterms="http://purl.org/dc/terms/"
+    xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" vc:minVersion="1.1"
+    xmlns="http://pubrouter.jisc.ac.uk/rioxxplus">
+    <xs:import namespace="http://www.rioxx.net/schema/v2.0/rioxxterms/"
+        schemaLocation="http://rioxx.net/schema/v2.0/rioxx/rioxxterms.xsd"/>
+    <xs:import namespace="http://ali.niso.org/2014/ali/1.0"
+        schemaLocation="http://rioxx.net/schema/v2.0/rioxx/ali.xsd"/>
+    <xs:import namespace="http://purl.org/dc/terms/"
+        schemaLocation="http://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd"/>
+
+    <xs:complexType name="contributor_type">
+        <xs:all>
+            <xs:element name="type" minOccurs="0" type="xs:string"/>
+            <xs:element name="id" minOccurs="0" maxOccurs="unbounded">
+                <xs:complexType>
+                    <xs:simpleContent>
+                        <xs:extension base="xs:string">
+                            <xs:attribute name="type" type="xs:string" use="required"/>
+                        </xs:extension>
+                    </xs:simpleContent>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="email" minOccurs="0" maxOccurs="unbounded" type="xs:string"/>
+            <xs:element name="surname" minOccurs="0" type="xs:string"/>
+            <xs:element name="firstnames" minOccurs="0" type="xs:string"/>
+            <xs:element name="suffix" minOccurs="0" type="xs:string"/>
+            <xs:element name="org_name" minOccurs="0" type="xs:string"/>
+        </xs:all>
+    </xs:complexType>
+
+    <xs:element name="entry">
+        <xs:complexType>
+            <xs:all>
+                <xs:element name="note" minOccurs="0" maxOccurs="unbounded" type="xs:string"/>
+                <xs:element name="comment" minOccurs="0" type="xs:string"/>
+                <xs:element name="relation" minOccurs="0" maxOccurs="unbounded">
+                    <xs:complexType>
+                        <xs:simpleContent>
```

Comment:
The extension is required if you want any content in the element.

Your type would validate this:
`<relation url="http://someurl.url" format="application/zip"></relation>`

but not this
`<relation url="http://someurl.url" format="application/zip">BAD VALUE</relation>`